src/main/java/com/robertx22/mine_and_slash/vanilla_mc/packets/interaction/IParticleSpawnMaterial.java

```diff
@@ -25,7 +29,7 @@ public interface IParticleSpawnMaterial {
     InteractionResultHandler.ParticleSpawnType getSpawnType();
     void spawnOnClient(Entity entity);
 
-    enum Type implements IParticleSpawnMaterial {
+    enum Type implements IParticleSpawnMaterial, IAutoLocName {
```

Comment:
If there's only like 2 or 3 things i tend to just use Words enum for it

Reply:
Honestly, I don't like to put things into Words unless they are really trivial or hard to categorize. But I'll do that.